Crashlytics/Crashlytics/Models/Record/FIRCLSReportAdapter.m

```diff
@@ -26,6 +26,8 @@
 #import <nanopb/pb_decode.h>
 #import <nanopb/pb_encode.h>
 
+const NSUInteger IMPORTANCE_FOR_EXCEPTION = 0;
```

Comment:
In FIRCLSRecordThread.m:21, we define IMPORTANCE_IN_CRASHED_THREAD. Does it make more sense to have this const be defined FIRCLSRecordFrame?

Reply:
Yeah that probably makes more sense to organize it all there